rules/gcs.bzl

```diff
@@ -0,0 +1,91 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Skylark rule for pulling a file from GCS bucket.
+"""
+
+_GCS_FILE_BUILD = """
```

Comment:
This looks a little bit odd to me... could you give me some reference why we need to do `_GCS_FILE_BUILD=...`?

Reply:
I referred to this: https://github.com/bazelbuild/bazel/blob/master/tools/build_defs/repo/http.bzl#L58